.github/ISSUE_TEMPLATE/z_release_major.md

```diff
@@ -42,17 +40,14 @@ If all is good, then in 24h, I will start the release. Thanks!
   - [ ] Create a new document for the release in the [releases folder](https://github.com/medic/cht-docs/tree/main/content/en/core/releases).
   - [ ] Ensure all issues are in the GH Milestone, that they're correctly labelled (in particular: they have the right Type, "UI/UX" if they change the UI, and "Breaking change" if appropriate), and have human readable descriptions.
```

Comment:
In this step, it says that we need to verify that all the tickets are correctly labeled, but what does `correctly labeled` mean? 
When I was at this step I assumed that all the tickets **must** have the label `type` and that they **might** have other labels depending on the change. If I am correct, can we update the step to be more specific, clarifying that all the tickets need to have at least the `type` tag and any other one that is appropriate?

Reply:
Done.